### DISCUSSION TITLE
IPv6 Address missing in the setup guide

### DISCUSSION THREAD

**Discussion by pattyland** [AdguardTeam/AdGuardHome#3014] [Q&A] [answered]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:**
  * v0.105.2
* **How did you install AdGuard Home:**
  * Home Assistant Addon
* **How did you setup DNS configuration:**
  * DHCP
* **If it's a router or IoT, please write device model:**
  * 
* **CPU architecture:**
  * x86
* **Operating system and version:**
  * Home Assistant OS

### Expected Behavior
View IPv6 Address in the setup guide

### Actual Behavior
IPv6 is missing on the setup guide since the last update

### Additional Information
I checked all options of AGH, IPv6 seems to be disabled nowhere.
Could be something at docker level I guess

**Comment by ainar-g** [comment]
Hi, did you enable IPv6 for your Docker daemon as well?

https://docs.docker.com/config/daemon/ipv6/

**Reply by pattyland** [reply]
I'm using Home Assistant OS, I'm not quite sure to be honest.
Wenn I ssh into the docker container from AGH and do run ifconfig, I still see the IPv6 addresses I configured all my devices for. So I would say yes?

You can view the exact configuration here: https://github.com/hassio-addons/addon-adguard-home

I guess it may be related to #1401 ?

**Comment by pattyland** [comment, marked as answer]
Update to v0.106 fixed that 👍
